content/blog/2023-09-27-epidatr.Rmd

```diff
@@ -0,0 +1,96 @@
+
+This package is designed to streamline the downloading and usage of data from the Delphi Epidata API. It provides a simple R interface to the API, including functions for downloading data, parsing the results, and converting the data into a tidy format. The API stores a historical record of all data, including corrections and updates, which is particularly useful for accurately backtesting forecasting models. We also provide packages for downstream data processing ([epiprocess](https://github.com/cmu-delphi/epiprocess)) and modeling ([epipredict](https://github.com/cmu-delphi/epipredict)).
+
+## Usage
+
+```R
+library(epidatr)
+# Obtain the smoothed covid-like illness (CLI) signal from the Facebook survey as it was on April 10, 2021 for the US
+epidata <- covidcast(
+  source = "fb-survey",
+  signals = "smoothed_cli",
+  geo_type = "nation",
+  time_type = "day",
+  geo_values = "us",
+  time_values = epirange(20210101, 20210601),
+  as_of = "2021-06-01")
+epidata
+```
+
+```
+# A tibble: 6 × 15
+  geo_value signal       source   geo_type time_type time_value
+  <chr>     <chr>        <chr>    <fct>    <fct>     <date>
+1 us        smoothed_cli fb-surv… nation   day       2021-04-05
+2 us        smoothed_cli fb-surv… nation   day       2021-04-06
+3 us        smoothed_cli fb-surv… nation   day       2021-04-07
+4 us        smoothed_cli fb-surv… nation   day       2021-04-08
+5 us        smoothed_cli fb-surv… nation   day       2021-04-09
+6 us        smoothed_cli fb-surv… nation   day       2021-04-10
+# ℹ 9 more variables: direction <dbl>, issue <date>,
+#   lag <int>, missing_value <int>, missing_stderr <int>,
+#   missing_sample_size <int>, value <dbl>, stderr <dbl>,
+#   sample_size <dbl>
+```
+
+## Installation
+
+You can install the stable version of this package from CRAN:
+
+``` r
+install.packages("epidatr")
+pak::pkg_install("epidatr")
+renv::install("epidatr")
+```
+
+Or if you want the development version, install from GitHub:
+
+``` r
+# Install the dev version using `pak` or `remotes`
+pak::pkg_install("cmu-delphi/epidatr")
+remotes::install_github("cmu-delphi/epidatr")
+renv::install("cmu-delphi/epidatr")
+```
+
+### API Keys
+
+The Delphi API requires a (free) API key for full functionality. To generate
+your key, register for a pseudo-anonymous account
+[here](https://api.delphi.cmu.edu/epidata/admin/registration_form) and see more
+discussion on the [general API
+website](https://cmu-delphi.github.io/delphi-epidata/api/api_keys.html). The
+`epidatr` client will automatically look for this key in the R option
+`delphi.epidata.key` or in the environment variable
+`DELPHI_EPIDATA_KEY`. We recommend storing your key in `.Renviron` file, which R
+will read by default.
+
+Note that for the time being, the private endpoints (i.e. those prefixed with
+`pvt`) will require a separate key that needs to be passed as an argument.
+
+## For users of the `covidcast` R package
```

Comment:
This formatting screws with the layout of the "outline" element (the pinkish sorta table of contents on the left side of the blog post)...  I think we can just remove the fixed-width formatting, but putting "covidcast" in quotes would work too.
```suggestion
## For users of the covidcast R package
```